demo/python/multiclient-remote-control/client1/client1.py

```diff
@@ -0,0 +1,80 @@
+"""The Python implementation of the GRPC Remote Attestation client."""
+
+from __future__ import print_function
+import logging
+
+import grpc
+import base64
+import securexgboost as xgb
+import argparse
+import os
+
+DIR = os.path.dirname(os.path.realpath(__file__))
+HOME_DIR = DIR + "/../../../../"
+username = "user1"
+
+def run(channel_addr, sym_key_file, priv_key_file, cert_file):
+    xgb.init_user(username, sym_key_file, priv_key_file, cert_file)
+
+    # Remote attestation
+    print("Remote attestation")
+    enclave_reference = xgb.Enclave(addr=channel_addr)
+    enclave_reference.attest(verify=False)
```

Comment:
Default checked in version should have `verify=True`

Reply:
Fixed